src/Workspaces/CSharp/Portable/Formatting/FormattingHelpers.cs

```diff
@@ -81,6 +81,9 @@ public static bool IsCloseParenInParameterList(this SyntaxToken token)
             return token.Kind() == SyntaxKind.CloseParenToken && token.Parent.Kind() == SyntaxKind.ParameterList;
         }
 
+        public static bool IsCloseParenInTuple(this SyntaxToken token)
+            => token.Kind() == SyntaxKind.CloseParenToken && token.Parent is TupleExpressionSyntax;
```

Comment:
```suggestion
            => token.IsKind(SyntaxKind.CloseParenToken) && token.Parent.IsKind(SyntaxKind.TupleExpression);
```

Reply:
Of course, I should have thought of this ;)